src/Engine/ProtoCore/Lang/TraceUtils.cs

```diff
@@ -5,6 +5,7 @@
 
 namespace ProtoCore.Lang
 {
+	[Obsolete("This class is not used anymore, use the same class from Dynamo.Services namespace",false)]
     public static class TraceUtils
```

Comment:
alignment?

Reply:
Also add a TODO to remove this class in 3.0.

Reply:
yes, will align this.